fast-markdown-mcp/src/fast_markdown_mcp/server.py

```
@@ -478,6 +698,32 @@ async def list_tools() -> list[types.Tool]:
                         "required": ["file_id"]
                     }
+                ),
+                types.Tool(
+                    name="smart_section_search",
+                    description="Advanced search for sections with ranking and confidence scores",
+                    inputSchema={
+                        "type": "object",
+                        "properties": {
+                            "query": {
+                                "type": "string",
+                                "description": "Search query to find in sections"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)